packages/front-end/components/Layout/SidebarLink.tsx

```diff
@@ -37,6 +39,10 @@ const SidebarLink: FC<SidebarLinkProps> = (props) => {
 
   const [open, setOpen] = useState(selected);
 
+  if (props.name === "Get Started" && !guidedOnboarding) {
+    return null;
+  }
```

Comment:
This is a generic component and shouldn't have a reference to one specific link or feature flag.

What we could do instead is add a new prop to `SidebarLinkProps` called `feature?: string`.  And if that is set, evaluate the feature flag and return `null` if it's off.

```tsx
const growthbook = useGrowthBook();

if (props.feature && growthbook.isOff(props.feature)) return null;
```

Then we can just set the` feature` prop in `Layout.tsx` for the Getting Started link.

I think we'll end up using this quite a bit when we launch new sections of the app.

Reply:
I like this pattern a lot, thanks for the recommendation.